docs/reference/query-dsl/query-string-syntax.asciidoc

```diff
@@ -116,14 +116,12 @@ Use with caution!
 [[query-string-fuzziness]]
 ====== Fuzziness
 
-We can search for terms that are
-similar to, but not exactly like our search terms, using the ``fuzzy''
-operator:
+You can run <<query-dsl-fuzzy-query,`fuzzy` queries>> using the `~` operator:
 
     quikc~ brwn~ foks~
 
-This uses the
-{wikipedia}/Damerau-Levenshtein_distance[Damerau-Levenshtein distance]
+For these queries, the query string is not <<analysis,analyzed>>. Instead, the
```

Comment:
For fuzzy queries, we will  [normalize](https://www.elastic.co/guide/en/elasticsearch/reference/current/analysis-normalizers.html#analysis-normalizers) a query string. This means, that we will take an analyzer for the field or the provided analyzer,  take only token filters that can normalize and run the query string through it.

Reply:
Thanks for the correction @mayya-sharipova. I pushed a few commits to fix this.